src/InputContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { updateRestaurantInfo, addRestaurant } from './actions';
+import RestaurantInput from './RestaurantInput';
+
+function InputContainer() {
+  const dispatch = useDispatch();
+
+  function handleChange(event) {
+    dispatch(updateRestaurantInfo({ [event.target.name]: event.target.value }));
+  }
```

Comment:
구조분해할당을 해줄 수 있을 것 같아요~

```suggestion
  function handleChange(event) {
    const { target: { name, value } } = event;
    dispatch(updateRestaurantInfo({ [name]: value }));
  }
```

Reply:
구조분해 할당 할까 말까 했었는데 하는게 낳았네요 ㅎ